app/views/layouts/application.html.erb

```diff
@@ -19,7 +19,6 @@
   <%= csrf_meta_tags %>
```

Comment:
Does the `<%= stylesheet_link_tag "application" %>` line (above) need an explicit `:media => "all"` adding, or is it okay?

Reply:
Ah probably should change it to be consistent with what we are doing elsewhere